graylog2-web-interface/src/components/sidecars/configuration-forms/ConfigurationVariablesHelper.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { Alert, Button, Table, Modal } from 'react-bootstrap';
+import { Spinner } from 'components/common';
+import { BootstrapModalConfirm } from 'components/bootstrap';
+import BootstrapModalWrapper from 'components/bootstrap/BootstrapModalWrapper';
+import CombinedProvider from 'injection/CombinedProvider';
+import EditConfigurationVariableModal from './EditConfigurationVariableModal';
+import ConfigurationHelperStyle from './ConfigurationHelper.css';
+
+const { ConfigurationVariableActions } = CombinedProvider.get('ConfigurationVariable');
+
+class ConfigurationVariablesHelper extends React.Component {
+  state = {
+    configurationVariables: [],
+    errorModalContent: {},
+  };
+
+  componentDidMount() {
+    this._reloadConfiguration();
+  }
+
+  _reloadConfiguration = () => {
+    ConfigurationVariableActions.all()
+      .then((configurationVariables) => {
+        this.setState({ configurationVariables: configurationVariables });
+      });
+  };
+
+  _getId = (idName, index) => {
```

Comment:
Do we need this method? We never pass an index to it, so I think we can remove it.